### PR TITLE
Robie

### PR DESCRIPTION
**Pull request by p4wix**

### CODE REVIEW

**Review by wojtkolos — COMMENTED** [review]
approved

**Comment by p4wix** [comment]
app

File: backend/routes/test_users.rest [hunk 1]
```diff
@@ -0,0 +1,29 @@
+GET http://localhost:3001/api/users
+
+###
+GET http://localhost:3001/api/users/
+
+###
+POST http://localhost:3001/api/users
```
Comment (p4wix) [inline backend/routes/test_users.rest:7]: do ziamny

**Review by mwegrzak — COMMENTED** [review]
git

**Review by Pacuch — APPROVED** [review]
very fantastic code, i'm such impressed with kunszt of this project, wish i could do more projects with those guys

**Comment by p4wix** [comment]
> very fantastic code, i'm such impressed with kunszt of this project, wish i could do more projects with those guys

thanks, your world are big

**Pull request merged**